doc/apiv3-documentation.apib

```diff
@@ -93,7 +93,7 @@ For the future we plan to add authentication modes based on OAuth2 and OpenID Co
 
 Links to other resources in the API are represented uniformly by so called link objects.
 
-## Properties
+## Local Properties
```

Comment:
If you want to be thorough: There are a few places like this we you either have a : appended and then some others without. You may want to consider making sure that its consistent across the entire document. This also applies to captions like Linked Properties.